src/modules/ekf2/EKF/aid_sources/gnss/gnss_checks.cpp

```diff
@@ -81,7 +81,7 @@ bool GnssChecks::run(const gnssSample &gnss, uint64_t time_us)
 
 bool GnssChecks::runSimplifiedChecks(const gnssSample &gnss)
 {
-	_check_fail_status.flags.fix = (gnss.fix_type < 3);
+	_check_fail_status.flags.fix = (gnss.fix_type < _params.ekf2_req_fix);
```

Comment:
```suggestion
	_check_fail_status.flags.fix = (gnss.fix_type < 3);
```

Those are the "simplified checks" and should have relaxed thresholds. The purpose is to let you select a tight limit (e.g:: RTK fix) in the `runInitialFixChecks`, and then the GNSS fusion will still not fail if the fix gets worse during the flight.

Reply:
Makes sense